app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()[1]
+    file_copy_name = command.split()[2]
```

Comment:
don't split your command twice
create separated variable which equals to command.split() and use it in the following computations

Reply:
Fixed